scripts/participateInAuction.js

```diff
@@ -37,7 +37,7 @@ async function participateInAuction(coordinatorAddress, network) {
 
       infoAuction("Successfully called participateInAuction! in tx: %s", result.tx)
     } else {
-      debugAuction("Can't participate in auction")
+      debugAuction("It's not time to participate in auction")
```

Comment:
Less excited about this one.

Reply:
I understand. But devops were yesterday trying to figure out why the pool was not working. And it was because they understood "Can't participate in auction" was an error, it was perceived as it tried and it failed.

I explained the meaning, but also rewrited removing the negative connotation.

Let me know if u prefer any other phrase